src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.java

```diff
@@ -32,6 +34,12 @@
  * </p>
  * <ul>
  * <li>
+ * Property {@code ignoreAnnotatedBy} - Ignore number of parameters for
+ * methods and constructors annotated with the specified annotation(s).
```

Comment:
Done.

Reply:
Had written in that way to be consistent with the description of the `ignoreOverriddenMethods` property of the check, now that we have changed it, should I do the same for `ignoreOverriddenMethods` perhaps in a minor:... pr after this one gets merged?

Reply:
> Had written it that way to be consistent with the description of `ignoreOverriddenMethods` property of the check, now that we have changed it, should I do the same for `ignoreOverriddenMethods` perhaps in a minor:... pr after this one gets merged?

@sktpy please restate this, I do not understand this question.

Reply:
The description of `ignoreOverriddenMethods` is as following:
https://github.com/checkstyle/checkstyle/blob/7e6b50ff6983d37e2b7235fc1be3adb7a525dd1e/src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ParameterNumberCheck.java#L86

This was the reason I had kept the description of `ignoreAnnotatedBy` in the similiar way, but since now it is changed as suggested to not have `...number of parameters for...`, My question was that should the description of `ignoreOverriddenMethods` be also changed in a similar way for the purpose of consistency? (after this is merged, in a minor pr)

Reply:
Ah! I understand now. Thanks for the explanation. Yes, we could send another PR with `doc:` prefix for this, no need to open an issue for a simple documentation update.